pyproject.toml

```diff
@@ -65,9 +65,8 @@ extractors = [
     "sonpy;python_version<'3.10'",
     "lxml", # lxml for neuroscope
     "scipy",
-    # ONE-api and ibllib for streaming IBL
-    "ONE-api>=1.19.1",
-    "ibllib>=2.21.0",
+    "ONE-api>=1.19.1", # alf sorter and streaming IBL
```

Comment:
Let's upgrade this to their latest versions. I will push this if you don't mind.